test/e2e/e2e_selenium_test.py

```diff
@@ -33,11 +47,15 @@ def driver():
 def test_task_journey(driver, test_app): 
     test_item_name = 'Finish selenium test'
     # this is to stop a connection issue that was causing intermittant failures
-    time.sleep(3)
+    time.sleep(5)
     driver.get('http://localhost:5000/')
+    driver.implicitly_wait(10)
```

Comment:
You only need to set this wait time once; after that it persists for the remainder of the session - whenever asked to find an element it will allow up to this amount of time before failing